numpy/_core/src/npysort/simd_qsort.dispatch.cpp

```diff
@@ -89,31 +71,32 @@ template<> void NPY_CPU_DISPATCH_CURFX(QSort)(double *arr, intptr_t size)
 {
     avx512_qsort(arr, size);
 }
-template<> void NPY_CPU_DISPATCH_CURFX(ArgQSort)(int32_t *arr, npy_intp *arg, npy_intp size)
+#elif USE_HIGHWAY
```

Comment:
```suggestion
```
Based on the proposed suggestion, the following functions in this modification should be defined within a separate new source `highway_qsort.dispatch.cpp`.